setup.py

```diff
@@ -298,6 +298,13 @@ def run_build():
             "Topic :: Software Development :: Compilers",
             "Topic :: Software Development :: Libraries :: Python Modules"
         ],
+        project_urls={
+            "Documentation": "https://cython.readthedocs.io/",
+            "Donate": "https://cython.readthedocs.io/en/latest/src/donating.html",
+            "Source code": "https://github.com/cython/cython",
+            "Bug tracker": "https://github.com/cython/cython/issues",
+            "User group": "https://groups.google.com/g/cython-users",
```

Comment:
mailing list would be a better name I think, although it's probably not the best, but an established one